cli/commands/extension/register.go

```diff
@@ -0,0 +1,49 @@
+package extension
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/client"
+	"github.com/sensu/sensu-go/types"
+	"github.com/spf13/cobra"
+)
+
+// RegisterCommand adds command that allows registering extensions
+func RegisterCommand(cli *cli.SensuCli) *cobra.Command {
+	return &cobra.Command{
+		Use:   "register NAME URL",
+		Short: "register extensions",
+		RunE:  runRegister(cli.Client, cli.Config.Organization()),
+	}
+}
+
+func runRegister(client client.APIClient, org string) func(*cobra.Command, []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		if len(args) != 2 {
```

Comment:
Do we want to allow the use of flags here? Just curious if that would be helpful!

Reply:
Ah I figure we can always change that if flags are added later. I don't see a use for them right now though.